README.md

```diff
@@ -1,25 +1,53 @@
-This is similar to React's `Create React App`. The idea is to provides a
+
+# Create Socket App 
+
+`Create Socket App` is similar to React's `Create React App`. The idea is to provides a
 few basic boilerplates and some strong opinions so you can get coding on a
 production quality app as quickly as possible.
 
-## Usage: 
+## Getting Started 
+
+Before getting started, you may need some `Prerequisites`, to learn more you can check
+[Socket Runtime](https://sockets.sh/).
+**You'll also need to have Node>= 16 on your local developement machine.**
+That's it, you are ready to go!
+
+To create a new app, you may choose one of the following methods: 
+
+With **npm**:
 
-With npm:
+Any of these following commands will work
 ```bash
+npx create socket-app [react | svelte | tonic | vanilla | vue]
+npm init socket-app [react | svelte | tonic | vanilla | vue]
 npm create socket-app [react | svelte | tonic | vanilla | vue]
+
 ```
 
-With yarn:
+https://user-images.githubusercontent.com/79177582/219162686-649bbdb9-015a-41bd-a613-af665a5a199e.mov  
+
+`Notes` : 
+
+`npx` is a package runner tool that comes with `npm 5.2` and above - [learn more here](https://blog.npmjs.org/post/162869356040/introducing-npx-an-npm-package-runner#:~:text=npx%20is%20a%20tool%20intended%20to%20help%20round,tools%20and%20other%20executables%20hosted%20on%20the%20registry.)
```

Comment:
hmm, we can focus only on our tool and don't let people think too much about other tools. We should use `npm init` or `npm create` IMO

Reply:
>hmm, we can focus only on our tool and don't let people think too much about other tools. We should use npm init or npm create IMO

@chicoxyzzy I'll make the change.